app/models/target.rb

```diff
@@ -0,0 +1,35 @@
+# == Schema Information
+#
+# Table name: targets
+#
+#  id         :bigint           not null, primary key
+#  latitude   :float            not null
+#  longitude  :float            not null
+#  radius     :float            not null
+#  title      :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  topic_id   :bigint           not null
+#  user_id    :bigint           not null
+#
+# Indexes
+#
+#  index_targets_on_topic_id                        (topic_id)
+#  index_targets_on_topic_id_and_title_and_user_id  (topic_id,title,user_id) UNIQUE
+#  index_targets_on_user_id                         (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (topic_id => topics.id)
+#  fk_rails_...  (user_id => users.id)
+#
+
+class Target < ApplicationRecord
+  belongs_to :topic
+  belongs_to :user
+
+  validates :latitude, :longitude, :radius, :topic_id, :user_id, presence: true
```

Comment:
It's not necessary to validate the presence of belgons_to associations, it's made by default in rails >= 5

Reply:
@gonzaM  does it?
my presence validation tests fail if I delete the presence validation on the model